common/src/main/java/org/apache/falcon/security/SecurityUtil.java

```diff
@@ -115,6 +115,16 @@ public static boolean isAuthorizationEnabled() {
                 "falcon.security.authorization.enabled", "false"));
     }
 
+    /**
+     * Checks if CSRF filter is enabled in the configuration.
+     *
+     * @return true if falcon.security.csrf.enabled is enabled, false otherwise
+     */
+    public static boolean isCSRFFilterEnabled() {
```

Comment:
This should also check if the security is enabled as without that there is no authentication.